lib/build/js/relink.js

```diff
@@ -49,8 +50,8 @@ var pathUtils = require('path');
 
                 if (file && file.jsRef)
                 {
-                  token[1] = ['dot', ['name', 'basis'], 'resource'];
-                  token[2] = [['string', './' + file.jsRef]];
+                  token.callee = at.parse('basis.resource', true);
```

Comment:
It's better replace `at.parse` into AST build function, since code here is predictable.
Something like that
```js
function buildPath(name){
  var result = {
    type: 'Identifier',
    name: name
  };

  for (var i = 1; i < arguments.length; i++)
    result = {
      type: 'MemberExpression',
      computed: false,
      object: result,
      property: {
        type: 'Identifier',
        name: arguments[i]
      }
    };

  return result;
}

buildPath('basis', 'resource')
```